### PR TITLE
Update collectioncharacteristics.md

### PR DESCRIPTION
**Pull request by Ragman22**
Hi @CobaltGray 
I added the SLED column and a note on SLED under the table. Please check if the stats are also correct.
Also, let me know if there's more needed.
BR

### CODE REVIEW

File: docs/productguide/collectioncharacteristics.md [hunk 1]
```diff
@@ -7,27 +7,28 @@ The following provides additional technical information on the performance of th
...
-| Performant Incidence Range [deg]  | 15-30    | 20-35     | 20-35  | 21-29   | [Note 12](#notes-and-explanations)   |
-| Time Dominant Incidence Range [deg] | 11-44  | 11-44     | 11-44 | N/A      | [Note 13](#notes-and-explanations)   |
+| PARAMETER      | STRIP        |  SPOT   | SPOT EXTENDED AREA |  SPOT EXTENDED DWELL   | SCAN          | COMMENTS                            |
+|----------------|--------------|---------|--------------------|------------------------|---------------|------------|
+| Product Short Name            | 'SM'         | 'SLH'     | 'SLEA' | 'SLED' | 'SC'    | [Note 1](#notes-and-explanations)    |
+| Radar Beams Used              |  1           | 1         | 1     | 1       | 4       | [Note 2](#notes-and-explanations)    |
+| Nominal swath width [km]      | 30           | 5         | 15    | 5       | 100     | [Note 3](#notes-and-explanations)    |
+| Nominal product length (Azimuth Direction) [km] | 50 | 5 | 15    | 5       | 100     | [Note 4](#notes-and-explanations)    |
+| Nominal collection duration [sec] | 10       | 10        | 10     | 10     | 15      |                                      |
```
Comment (CobaltGray) [inline docs/productguide/collectioncharacteristics.md]: Should be 25 sec

File: docs/productguide/collectioncharacteristics.md [hunk 2]
```diff
@@ -7,27 +7,28 @@ The following provides additional technical information on the performance of th
...
+| Maximum collection duration [sec] | 35-75    | N/A       | N/A   | 75      | N/A     | [Note 5](#notes-and-explanations)    |
```
Comment (CobaltGray) [inline docs/productguide/collectioncharacteristics.md]: Should be 25sec

File: docs/productguide/collectioncharacteristics.md [hunk 3]
```diff
@@ -7,27 +7,28 @@ The following provides additional technical information on the performance of th
...
+| Maximum Scene Length [km]         | 500  | 5         | 15     | 5         | 500      | [Note 5](#notes-and-explanations)    |
+| Noise Equivalent Sigma-Zero [ dBm^2^/m^2^ ]  | -21.5 to -20 | -18 to -15 | -18 to -15 | -18 to -15 | -22.2 to -21.5 | [Note 6](#notes-and-explanations)    |
+| Azimuth Ambiguity Ratio [dB]  | -17          | -17       | -17    | -17       | -17     |                                      |
+| Range Ambiguity Ratio [dB]    | <-20          | <-20       | <-20    | <-20       | <-20     |                                      |
+| Geospatial Accuracy [m RMSE] | 6            | 6         | 6      | 6         | 15      |  [Note 14](#notes-and-explanations) |
+| ESA Copernicus Contributing Mission (CCM) Class[@copernicusClass] | VHR2 | VHR1 | VHR1 | VHR1 | HR1 |                     |
+| Polarization                  | VV           | VV        | VV     | VV        | VV      |                                      |
+| RNIIRS                        | 3.6          | 5.5       | 5.5    | 5.5       | 2.1     | [Note 8](#notes-and-explanations)    |
```
Comment (CobaltGray) [inline docs/productguide/collectioncharacteristics.md]: should be 6.5

File: docs/productguide/collectioncharacteristics.md [hunk 4]
```diff
@@ -7,27 +7,28 @@ The following provides additional technical information on the performance of th
...
+| RGIQE [bits/$m^2$]               | 0.8          | 22        |  8.4     | 22        | 0.1     | [Note 9](#notes-and-explanations)    |
```
Comment (CobaltGray) [inline docs/productguide/collectioncharacteristics.md]: Should be 125

**Review by CobaltGray — CHANGES_REQUESTED** [review]
Could you please make the changes I've highlighted. Might as well make those numbers correct. (based on current Gen2 SAR model at 35deg incidence)

File: docs/productguide/collectioncharacteristics.md (hunk 1 above)
Comment (Ragman22) [reply docs/productguide/collectioncharacteristics.md]: Thx! Updated!

File: docs/productguide/collectioncharacteristics.md (hunk 2 above)
Comment (Ragman22) [reply docs/productguide/collectioncharacteristics.md]: Thx! Updated!

File: docs/productguide/collectioncharacteristics.md (hunk 3 above)
Comment (Ragman22) [reply docs/productguide/collectioncharacteristics.md]: Thx! Updated!

File: docs/productguide/collectioncharacteristics.md (hunk 4 above)
Comment (Ragman22) [reply docs/productguide/collectioncharacteristics.md]: Thx! Updated!

**Review by CobaltGray — APPROVED** [review]
Looks great - thanks

**Pull request merged**